ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/snap/RetryingGetAccountRangeFromPeerTask.java

```diff
@@ -28,7 +28,7 @@
 import org.apache.tuweni.bytes.Bytes32;
 
 public class RetryingGetAccountRangeFromPeerTask
-    extends AbstractRetryingPeerTask<AccountRangeMessage.AccountRangeData> {
+    extends AbstractRetryingSwitchingPeerTask<AccountRangeMessage.AccountRangeData> {
```

Comment:
FYI making this a peer switching task will break the snap server check logic that @pinges uses in `SnapServerChecker`

Reply:
NVM - the peer checker code uses the non-retrying version, GetAccountRangeFromPeerTask